Patch python import logging

benjaminpkane opened the pull request:
Import logging is broken in v0.7.3 through v0.7.3.3 and fails silently due to a bare except.

This patch fixes the bug, and replaces the bare except with network exception errors only. I have added a test case as well so logging can be tested throughout development.

My mistake for not catching this earlier. The original bare except was of course inappropriate, as well.

Publishing patch release v0.7.3.4 now.

brimoor reviewed (approved):
Ah my bad, looks like I introduced this when I moved this method to the `uid` module. :(

benjaminpkane commented:
> Ah my bad, looks like I introduced this when I moved this method to the `uid` module. :(

I also missed it in review and did not verify in pre-releases so I will definitely own this one. And the bare except.

Anyways, we should catch things earlier now. And App analytics are look fine.
